### ISSUE TITLE
Can we add Features / Epics too?

### ISSUE THREAD

**Issue by gabrielmccoll** [ipatalas/vscode-sprint-planner#10]
Just to say I love this extension and if I could plan out a whole large project with a Yaml style syntax I'd be v happy.  
Sadly I don't think I have the skills to help code anything.

**Comment by ipatalas** [comment]
Yeah, that would be really cool but I don't have much time recently for that project so can't promise anything.

Can you try preparing a sample document with some fake epics, features and user stories inside so that it looks clean?

**Comment by ipatalas** [comment]
BTW can you describe your workflow a little bit? It seems odd to me to assign Epics and Features to iterations.

**Comment by gabrielmccoll** [comment]
Massive apology for not getting back.  I was thinking just in general about filling the backlog for long term planning. not just the current sprint.   My new job is on prem ADO so I don't think this works for me anymore anyway. Sad times as I love the project.

**Comment by ipatalas** [comment]
Hey, no worries. I do have a plan to introduce support for Epics/Features but that's not gonna happen overnight and that project is not my priority at the moment. 
Thanks for opinion. Maybe one day you'll be back in the game :)

**Comment by ReneRebsdorf** [comment]
@ipatalas I am currently looking into building a tool to create "backlog-as-code" and stumbled over your repository, looking for a decent Azure DevOps Extension to save me the work.
The idea would be to create backlogs, and in the future potentially sync the two.

Your structure and idea in this extension seems neat, and this Feature Request combined with [issue 14](https://github.com/ipatalas/vscode-sprint-planner/issues/14) would do the trick.

How is your perspective on increasing the scope of your tool to cover creating an entire backlog? I see @gabrielmccoll first suggested it in nov '19

**Comment by ipatalas** [comment]
I really wish I had time for this :( 
For the past year or so I wasn't even using the extension myself because our project was managed in a different way and it was no use for us.
I am starting a new project soon and it will be in Azure DevOps so hopefully I will start using it again.
I would like this extension to be a comprehensive tool but life is life :) I can no longer afford to spend my private time on that. That may and hopefully will change in future but for now I have too many other things to handle.

**Comment by ReneRebsdorf** [comment]
> I really wish I had time for this :(
> For the past year or so I wasn't even using the extension myself because our project was managed in a different way and it was no use for us.
> I am starting a new project soon and it will be in Azure DevOps so hopefully I will start using it again.
> I would like this extension to be a comprehensive tool but life is life :) I can no longer afford to spend my private time on that. That may and hopefully will change in future but for now I have too many other things to handle.

I think everyone can relate to your situation. Thank you for getting back so quickly. 

Best wishes
